aws-solutions/digital-evidence-archive-on-aws
# chore: upgrade rush and dependencies

- upgrade to latest rush stack versions
- update all (except onetable) dependencies with patch and minor version releases
- update selective major dependency versions and resolve minor issues
- always use an axios client


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
